core/src/style/textStyle.cpp

```diff
@@ -175,10 +174,9 @@ void TextStyle::onBeginDrawFrame(const View& _view, const Scene& _scene) {
         m_shaderProgram->setUniformMatrix4f("u_proj", glm::value_ptr(_view.getOrthoViewportMatrix()));
         m_dirtyViewport = false;
     }
+    
+    Style::onBeginDrawFrame(_view, _scene);
```

Comment:
TextStyle does not defines a material and text shaders do not define u_zoom uniform, calling Style::onBeginDrawFrame will also call these.